source/brailleTables.py

```diff
@@ -87,7 +98,9 @@ def listTables():
 	"UEBC-g2.ctb":"en-ueb-g2.ctb",
 }
 
-# Add builtin tables.
+
+# Add the builtin tables at import time.

```

Comment:
It also has been reverted to ease blaming. There of course are means to exclude revisions when blaming (and we'll need it if eventually linting the whole source tree), but I've thought it would be much easier this way.
Please note that in the current proposal, `terminate` does restore the module in the state it was before calling `initialize`.
Both methods really do `initialize` / `terminate` only the handling of custom tables.

@feerrenrut, @leonardder may we have your thoughts here before rewriting this part yet another time?

Reply:
I prefer leaving this as is and doing this in a follow up pr.

Reply:
It appears that the consensus is to leave as is.